Description: Make layout/xul/base/src/crashtests/395609.xul not require
 network access
Author: Chris Coulson <chris.coulson@canonical.com>
Forwarded: no

Index: firefox-trunk-22.0~a1~hg20130307r124110/layout/xul/base/src/crashtests/395609.xul
===================================================================
--- firefox-trunk-22.0~a1~hg20130307r124110.orig/layout/xul/base/src/crashtests/395609.xul	2013-03-08 02:16:28.000000000 +0000
+++ firefox-trunk-22.0~a1~hg20130307r124110/layout/xul/base/src/crashtests/395609.xul	2013-03-09 00:53:54.201591468 +0000
@@ -10,7 +10,7 @@
 </box>
 </box>
 <box style="position: fixed; ">
-  <editor src="http://google.com/"/>
+  <editor src="http://localhost/"/>
 </box>
 </box>
-</window>
\ No newline at end of file
+</window>
